packages/schematics/angular/guard/index.ts

```diff
@@ -37,13 +36,14 @@ export default function (options: GuardOptions): Rule {
 
     let implementations = '';
     let implementationImports = '';
-    if (options.implements.length > 0) {
-      implementations = options.implements.join(', ');
-      implementationImports = `${implementations}, `;
-      // As long as we aren't in IE... ;)
-      if (options.implements.includes('CanLoad')) {
-        implementationImports = `${implementationImports}Route, UrlSegment, `;
-      }
+    if (options.implements.length === 0) {
```

Comment:
I think the actual problem is within the schema.

I think the `implements` should be something along;
```
    "implements": {
      "type": "array",
      "description": "Specifies which interfaces to implement.",
      "uniqueItems": true,
      "enum": [
        "CanActivate",
        "CanActivateChild",
        "CanLoad"
      ],
      "default": "CanActivate",
      "x-prompt": {
        "message": "Which interfaces would you like to implement?",
        "type": "list",
        "multiselect": true,
        "items": [
          "CanActivate",
          "CanActivateChild",
          "CanLoad"
        ]
      }
    }
```

PS: the commit should probably be a `fix`

Reply:
The logic here see